woocommerce/google-listings-and-ads
# Add age group and adult product attributes

### Changes proposed in this Pull Request:



Part of #16 .

This PR adds age group and adult product attributes. These attributes are available for simple products and variations (for variable products, each variation has its own input for the age group attribute but "adult" can be set on a variable too).

**Note:** I'm not sure if we want to display the adult attribute for all products. Maybe we could have a global option that can be set for the store and only then show this attribute?

### Detailed test instructions:

1. Create a new product or edit an existing one
2. If it's a simple product: change the values of Age Group and Adult in the "Google Listings and Ads" tab in the product data section
3. If it's a variable product: change the values of Age Group and Adult for each variation in the "Google Listings and Ads" section under each variation. You can also set the Adult attribute for the variable product (parent) which will apply to all variations.
4. Save the product and make sure it's synced to Google
5. Make sure Age Group and Adult are correctly updated in Google Merchant Center


### Changelog Note:

> Tweak - Add Age Group and Adult attributes
